README.md

```diff
@@ -1,15 +1,72 @@
+
+The submariner operator installs the submariner components on a Kubernetes cluster.
 
 It's available on [OperatorHub:submariner](https://operatorhub.io/operator/submariner).
 
-The current release is very basic, and will deploy the submariner components
-in your cluster, but it does not deploy the broker yet.
+# Quickstart
+
+
+## Prerequisites
+
+Submariner has a few requirements to get started:
+
+- At least 2 Kubernetes clusters, one of which is designated to serve as the central broker that is accessible by all of your connected clusters; this can be one of your connected clusters, but comes with the limitation that the cluster is required to be up to facilitate interconnectivity/negotiation
+- Different cluster/service CIDR's (as well as different Kubernetes DNS suffixes) between clusters. This is to prevent traffic selector/policy/routing conflicts.
+- Direct IP connectivity between instances through the internet (or on the same network if not running Submariner over the internet). Submariner supports 1:1 NAT setups but has a few caveats/provider-specific configuration instructions in this configuration.
+- Knowledge of each cluster's network configuration
+- Worker node IPs on all the clusters must be outside of the cluster/service CIDR ranges.
+- kubefedctl installed ([0.1.0-rc3](https://github.com/kubernetes-sigs/kubefed/releases/tag/v0.1.0-rc3)).
+- kubectl installed.
+
+An example of three clusters configured to use with Submariner would look like the following:
+
+| Cluster Name | Provider | Cluster CIDR | Service CIDR | DNS Suffix |
+|:-------------|:---------|:-------------|:-------------|:-----------|
+| broker       | AWS      | 10.42.0.0/16 | 10.43.0.0/16 | cluster.local |
+| west         | vSphere  | 10.0.0.0/16  | 10.1.0.0/16  | west.local |
+| east         | AWS      | 10.98.0.0/16 | 10.99.0.0/16 | east.local |
+
+## Installation
+
+### Download
+Download the latest binary for your operating system and architecture, then install it on your path. The binary has no external dependencies.
+
+To install for Linux amd64 it in `~/.local/bin`:
+
+    mkdir -p ~/.local/bin
+    wget https://github.com/submariner-io/submariner-operator/releases/download/v0.0.3/subctl-v0.0.3-linux-amd64
+    install subctl-v0.0.3-linux-amd64 ~/.local/bin/subctl
+    rm subctl-v0.0.3-linux-amd64
+If `~/.local/bin` is on your PATH, you will then be able to run subctl.
+
+## Deployment
+
+To deploy submariner in a two Kubernetes cluster setup follow the below steps
+
+### Deploy Broker with Dataplane
 
-# Working on the operator
+To deploy the Submariner broker along with the data cluster run
+```
+./bin/subctl deploy-broker --kubeconfig <PATH-TO-KUBECONFIG-BROKER> dataplane --service-discovery --broker-cluster-context <BROKER-CONTEXT-NAME>
+```
+
+The --service-discovery and --broker-cluster-context  shall be skipped if the service discovery component, Lighthouse, is not required.
+
```

Comment:
This should also mention that we need `kubefedctl` on the path (and the appropriate version).

Reply:
I would add an specific section for the service discovery, where we move all the specific bits:

  * the warning on project status
  * the need of kubefedctl and kubectl
  * the flags.

I think it's better to avoid people accidentally installing discovery if they don't know what they are doing.

Reply:
+1 to Miguel's suggestion. Add a separate section for service discovery and cover everything under it.

Reply:
+1

Reply:
Done.